integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/predicate/NamedPredicateBaseIT.java

```diff
@@ -150,6 +150,18 @@ public void unknownPredicate_valueField() {
 				.hasMessageContainingAll( "Cannot use 'predicate:named:unknown-predicate' on field 'field1'" );
 	}
 
+	@Test
+	public void missingParam() {
+		assertThatThrownBy( () -> {
+			index.query()
+					.where( f -> f.named( "match-both-fields" )
+							.param( "value1", WORD_1 ) );
+		} )
+				.isInstanceOf( SearchException.class )
+				.hasMessageContainingAll(
+						"Param with name 'value2' has not been defined for the named predicate 'match-both-fields'." );
+	}
+
```

Comment:
Wouldn't it make more sense to put this test next to `org.hibernate.search.integrationtest.backend.tck.search.predicate.NamedPredicateProviderContextIT#param_nullName`?